src/Theming/ThemeSwitcher.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+import * as React from "react";
+import defaultTheme from "orbit-design-token";
+
+import ThemeProvider from "./ThemeProvider";
+
+type ThemeProps = {
+  children: any,
```

Comment:
children is not definitely any :)

Reply:
nice catch. just hint for better code reviews, if you know the solution it is good to give some hint. Something like:
Don't use `any` for children, React children have type `Node` see https://flow.org/en/docs/react/children/

```
import type { Node } from "react"
...
children: Node,
...
```

Reply:
OK, Fixed